tests/api/test_graphql.py

```diff
@@ -240,7 +240,7 @@ def test_filter_by_query_param(qs):
     for q in test_kwargs:
         q_objects |= Q(**{q: test_kwargs[q]})
     # FIXME: django 1.11 fails on called_once_with(q_objects)
```

Comment:
Can you check if it's still an issue with 2.2?

Reply:
It's still an issue. I wasn't sure about it being accidental.
Wouldn't be better to do a `@pytest.mark.xfail` or something? So we could provide a reason and see whenever it gets fixed? I'm unsure how we could do that cleanly... It seems odd to let that condition alone, it will get forgotten 😕